subscriptions/reporting/subscription_reports.rst

```diff
@@ -0,0 +1,129 @@
+=================================
+Discover our subscription reports
+=================================
+
+As we now know well enough, understanding how our business is going and where it is going is the key
+to success. This is even more the case when we offer subscription services or products.
+
+Before getting to the heart of the matter, it is very important to recall certain concepts essential
+to the proper understanding of the following reports:
+
+- **Monthly Recurring Revenue (MRR)**: MRR is arguably the most important metric for subscription
+  businesses. Indeed, it shows the revenue earned with subscription-based products or services every
+  single month. It is a consistent number that you can use to track all your recurring revenue over
+  time, in monthly increments.
+
+- **Annual Run-Rate (ARR)**: ARR is the yearly version of MRR. Based on the current MRR, you can
+  estimate the coming year's performance with the Annual Run-Rate. However, this estimation does not
+  take variations and growth into account.
+
+.. image:: media/difference-between-MRR-and-ARR.png
+  :align: center
+  :alt: Difference between MRR and ARR in Odoo Subscriptions
+
+- **Non-Recurring Revenue (NRR)**: NRR shows the revenue earned for everything else than
+  subscription-based products or services. This includes gains of a rare or unique nature that are
+  unlikely to occur in the ordinary course of businesses.
+
+- **Customer Retention**: This concept consists of engaging existing customers to continue buying
+  products or services from your business. As you may know, customer retention is quite a challenge
+  because you must prove yourselves worthy of your customers' trust.
```

Comment:
```suggestion
- **Customer Retention**: Practices to engage existing customers to continue buying
  products or services from your business. Customer retention can be a challenge, because you must prove you are worthy of your customers' trust.
```